source/mbed_trace.c

```diff
@@ -16,6 +16,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <stdarg.h>
+#include <inttypes.h>
```

Comment:
Inside the c-file, no. It's only a potential problem in the header.

@SeppoTakalo He's referring to the issue this is meant to avoid: https://github.com/ARMmbed/mbed-os/blob/master/features/FEATURE_COMMON_PAL/nanostack-libservice/mbed-client-libservice/ns_types.h#L49-L54